components/lib/crash/src/test/java/mozilla/components/lib/crash/prompt/CrashReporterActivityTest.kt

```diff
@@ -38,13 +40,18 @@ class CrashReporterActivityTest {
 
         val activity = Robolectric.buildActivity(CrashReporterActivity::class.java, intent)
             .create()
+            .visible()
             .start()
             .resume()
             .get()
 
         val closeButton = activity.findViewById<Button>(R.id.closeButton)
         closeButton.performClick()
 
+        runBlocking {
+            delay(100)
+        }
```

Comment:
Applying the changes in the comment above would give us an internal `job` field which we could `join`?
https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-coroutine-scope/index.html

Reply:
That's nice! 👍